packages/core/test/ServerTest.ts

```diff
@@ -612,6 +612,48 @@ class WoTServerTest {
         expect(thing).to.have.property("@context").to.deep.include({"@language": "xx"});
     }
 
+    @test async "should reject reading property without uriVariables when they are mandatory"() {
```

Comment:
Thank you @fatadel . You are right this is really an issue.
I will fix it in another PR to better keep track.

May I ask you to remove the test case in this pr. Sorry indeed and thanks!

Reply:
Sure, no worries!